File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Joaquin/Intake/IntakeConfig.java
package org.firstinspires.ftc.teamcode.Joaquin.Intake;

import com.qualcomm.robotcore.hardware.DcMotorSimple;


public class IntakeConfig{

    private final String servoId;
    
    private final DcMotorSimple.Direction servoDirection;

    public IntakeConfig(String servoId, DcMotorSimple.Direction servoDirection) {
        this.servoId = servoId;
        this.servoDirection = servoDirection;
    }

    public String getServoId() {
        return servoId;
    }
    
    public DcMotorSimple.Direction getServoDirection() {
        return servoDirection;
    }
}

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/arm/Intake/IntakeConfig.kt
package org.firstinspires.ftc.teamcode.arm.Intake

import com.qualcomm.robotcore.hardware.DcMotorSimple

data class IntakeConfig (
    val servoId: String,
    val servoDirection: DcMotorSimple.Direction,
)

val intakeConfig = IntakeConfig (
    servoId = "gripperServo", // Check the set name in the driver station
    servoDirection = DcMotorSimple.Direction.FORWARD, //Check the actual direction
)